mock_event.json

```diff
@@ -0,0 +1,186 @@
+{
+    "ref": "refs/heads/feature",
+    "before": "b40c4ce16063a5736bfd70f1789d8895e877fed1",
+    "after": "8bcdcf40ef2bde59cc907da5cf88a112b7c55de5",
+    "repository": {
+        "id": 596708517,
+        "node_id": "R_kgDOI5EMpQ",
+        "name": "test_pr",
+        "full_name": "syednabilashraf/test_pr",
+        "private": false,
+        "owner": {
+            "name": "syednabilashraf",
+            "email": "61734391+syednabilashraf@users.noreply.github.com",
+            "login": "syednabilashraf",
+            "id": 61734391,
+            "node_id": "MDQ6VXNlcjYxNzM0Mzkx",
+            "avatar_url": "https://avatars.githubusercontent.com/u/61734391?v=4",
+            "gravatar_id": "",
+            "url": "https://api.github.com/users/syednabilashraf",
+            "html_url": "https://github.com/syednabilashraf",
+            "followers_url": "https://api.github.com/users/syednabilashraf/followers",
+            "following_url": "https://api.github.com/users/syednabilashraf/following{/other_user}",
+            "gists_url": "https://api.github.com/users/syednabilashraf/gists{/gist_id}",
+            "starred_url": "https://api.github.com/users/syednabilashraf/starred{/owner}{/repo}",
```

Comment:
[mock_event.json, line 24]: The send_response method is called with a hardcoded value of 200. It is better to use one of the pre-defined constants from the http.server module, such as HTTPStatus.OK, instead of a hardcoded value.